enrolments/tests/views/test_classes.py

```diff
@@ -0,0 +1,87 @@
+from django.urls import reverse
+from rest_framework import status
+from rest_framework.test import APITestCase
+
+from accounts.models import User
+from enrolments.models import Session, Class
+from enrolments.serializers import (
+    SessionSerializer,
+    ClassListSerializer,
+    SessionDetailsSerializer,
+)
+
+
+class SessionTestCase(APITestCase):
```

Comment:
can we rename this file to `test_session_classes.py`, and this class `SessionClassesTestCase`? we'll have a view for classes later, so it'll be a bit cleaner down the road!

Reply:
renamed!